src/components/ProjectListItem.tsx

```diff
@@ -56,8 +61,16 @@ export const ProjectListItem: React.FC<ProjectListItemProps> = ({
                 ))}
             </TableCell>
         </TableRow>
-    </Link>
-);
+    );
+
+    return href ? (
```

Comment:
I think this logic must be in place of usage ProjectListItem. It would be if you can create issue about that 🙏🏼

Reply:
https://github.com/taskany-inc/issues/issues/1135